packages/google-oauth/google_server.js

```diff
@@ -17,7 +17,7 @@ function getServiceDataFromTokens(tokens) {
     scope: scopes
   };
 
-  if (hasOwn.call(tokens, "expiresAt")) {
+  if (hasOwn.call(tokens, "expiresIn")) {
```

Comment:
Should we keep supporting `expiresAt` as well?

Reply:
Good call - I think we should. Changes coming - thanks!

Reply:
Hmm - thinking about this more @benjamn, `expires_in` is the only supported option in the oauth token response from Google (based on the token endpoint we're using - https://accounts.google.com/o/oauth2/token). So since `expires_at` can no longer come back in the response, I don't think we need to support it. We won't be breaking backwards compatibility by dropping it since Google's endpoint will no longer return `expires_at`.

Reply:
This sounds correct to me.  Thanks for looking into this.